rllib/tests/test_checkpoint_restore.py

```diff
@@ -144,14 +146,14 @@ def valid_tf_checkpoint(checkpoint_dir):
 if __name__ == "__main__":
     failures = []
     for use_object_store in [False, True]:
-        for name in [
-                "SAC", "ES", "DQN", "DDPG", "PPO", "A3C", "APEX_DDPG", "ARS"
-        ]:
+        for name in ["DDPG"]:
             test_ckpt_restore(use_object_store, name, failures)
 
     assert not failures, failures
     print("All checkpoint restore tests passed!")
 
+    quit()
```

Comment:
Revert the changes in this file for merge?

Reply:
Ups, sorry, thanks for catching this!

Reply:
Done.